src/main/java/frc/robot/VisionProcessor.java

```diff
@@ -46,23 +50,32 @@
     public static double LATERAL_DISTANCE_FACTOR = 1.0;
     public static double PERPENDICULAR_DISTANCE_FACTOR = 0.6178;
     public static int CONSOLE_REPORTING_INTERVAL_MILLIS = 1000;
-    public static final boolean SWITCH_ONE_RECTANGLE_IS_SUFFICIENT = true;
+    public static double UNKNOWN = 99999;
+    public static final boolean SWITCH_ONE_RECTANGLE_IS_SUFFICIENT = false;
     private GripPipeline parent;
     private double averageArea = 0.0;
     private double averageDistanceToCenter = 0.0;
     private double distanceFromTarget= 0.0;
     private double lateralDistance = 0.0;
+    private int sizeSelected = 0;
     private double pixelPerInch = 0.0;
     private Mat lastFrame = null;
     public boolean foundTarget = false;
```

Comment:
do you still need foundTarget? that's what targetLock.isLocked() is for now right?